snuba/api.py

```diff
@@ -394,9 +391,7 @@ def parse_and_run_query(dataset, validated_body, timer):
         'sample': body.get('sample'),
     }
 
-    return util.raw_query(
-        validated_body, sql, clickhouse_ro, timer, stats
```

Comment:
I always interpreted `validated_body` here as the raw body provided by the client, and just realized that was wrong, since this is the value of the body *after* splitting. This changes the meaning of the value that is [logged as part of `raw_query`](https://github.com/getsentry/snuba/blob/de26a44af9648ce9059af43b540f53c7e8ec2a25/snuba/util.py#L490), which I think is OK for a couple reasons:

a.) I don't know that anybody actually looks at this value with any regularity
b.) We should change it later to be pre-split so it's actually representative of the client request